setup.py

```diff
@@ -6,7 +6,8 @@
     "chiabip158==1.0",  # bip158-style wallet filters
     "chiapos==1.0.1",  # proof of space
     "clvm==0.9.6",
-    "clvm_rs==0.1.6",
+    "maturin",
+    "clvm_rs@git+https://github.com/Chia-Network/clvm_rs.git@main#egg=clvm-rs",  # noqa
```

Comment:
We can't do this in something going to pypi, unfortunately we have to do a real release of `clvm_rs`.

Reply:
Oh, okay, you're aware.